indexers/src/services/relayer.service.ts

```diff
@@ -89,38 +83,42 @@ export class RelayerService {
   }
 
   /**
-   * Computes relayer specific metrics from the handlePostRequest transaction
+   * Computes relayer specific stats from the PostRequestHandled/PostResponseHandle event's transaction
    */
-  static async handlePostRequestTransaction(
-    transaction: HandlePostRequestsTransaction,
+  static async handlePostRequestOrPostResponseHandledEvent(
+    relayer_id: string,
+    transaction: EthereumTransaction<EthereumResult>,
     chain: SupportedChain,
   ): Promise<void> {
-    const { to, receipt, gasPrice } = transaction;
-    const { status, gasUsed } = await receipt();
+    const { status, gasUsed, effectiveGasPrice } = await transaction.receipt();
+    const ethPriceInUsd = await getCurrentEthPriceInUsd();
```

Comment:
Think we should call this native currency price

Reply:
Done